doc/experimental/phase1/migrate-calico.md

```diff
@@ -0,0 +1,48 @@
+# Migrate Calico
+
+### Version: v0.0.1-draft
+
+Migrate Calico 3.27.5 install to be managed by Helm
+
+## Modify the Tigera Operator objects to be annotated as being managed by Helm
+
+1.x Calico default cni installation through Kubernetes module is not deployed using a Helm Chart.
+The installed version of Calico needs to be transfomed to be manageable by Helm.
+
+tigera-operator:
+```text
+kubectl -n tigera-operator label ServiceAccount tigera-operator app.kubernetes.io/managed-by=Helm
+kubectl -n tigera-operator annotate ServiceAccount tigera-operator meta.helm.sh/release-name=mycalico
```

Comment:
Can a OCNE 1.x installation have calico module with a name other than `mycalico`? https://docs.oracle.com/en/operating-systems/olcne/1.9/calico/install.html mentions `mycalico` but it just an example - a customer could choose a different name in their OCNE 1.x installation.